docs/documentation/ko/release-notes/TypeScript 4.1.md

```diff
@@ -0,0 +1,655 @@
+---
+title: TypeScript 4.1
+layout: docs
+permalink: /ko/docs/handbook/release-notes/typescript-4-1.html
+oneline: TypeScript 4.1 Release Notes
+---
+
+## 템플릿 리터럴 타입 (Template Literal Types)
+
+Typescript에서 string 리터럴 타입은 특정한 문자열의 집합을 예상하는 함수들과 API들을 모델링할 수 있게 합니다.
+
+```ts twoslash
+// @errors: 2345
+function setVerticalAlignment(location: "top" | "middle" | "bottom") {
+  // ...
+}
+
+setVerticalAlignment("middel");
+```
+
+이것은 string 리터럴 타입이 기본적으로 string 값에 대해 맞춤법 검사를 할 수 있다는 점에서 꽤 괜찮습니다.
+또한 매핑된 타입에서 프로퍼티 이름으로서 string 리터럴을 사용할 수 있다는 점에서도 장점을 가집니다.
+
+이러한 면에서, string 리터럴을 건축 블럭(building blocks)들처럼 사용할 수 있습니다.
```

Comment:
위에 라인 문장이랑 바로 이어지도록 수정되어야 할 것 같습니다.

- 원문
> This is pretty nice because string literal types can basically spell-check our string values.

We also like that string literals can be used as property names in mapped types.
In this sense, they're also usable as building blocks:

Reply:
수정했습니다.